test/tests/meshgenerators/output_intermediate_mesh/output_intermediate_mesh.i

```diff
@@ -0,0 +1,34 @@
+[Mesh]
+  [./left]
```

Comment:
We should enable moose input formatting in your vs code. This `[./xxx]` and `[../]` syntax is old and we don't use it anymore. The input formatter will do this for you.